libfreerdp/core/multitransport.c

```diff
@@ -63,13 +63,20 @@ state_run_t multitransport_recv_request(rdpMultitransport* multi, wStream* s)
 
 	UINT32 requestId;
 	UINT16 requestedProto;
+	UINT16 reserved;
 	const BYTE* cookie;
 
 	Stream_Read_UINT32(s, requestId);      /* requestId (4 bytes) */
 	Stream_Read_UINT16(s, requestedProto); /* requestedProtocol (2 bytes) */
-	Stream_Seek(s, 2);                     /* reserved (2 bytes) */
+	Stream_Read_UINT16(s, reserved);       /* reserved (2 bytes) */
 	cookie = Stream_ConstPointer(s);
 	Stream_Seek(s, RDPUDP_COOKIE_LEN); /* securityCookie (16 bytes) */
+	if (reserved != 0)
+	{
+		WLog_WARN(TAG, "reserved is %d instead of 0, skipping %" PRIuz "bytes of unknown data",
```

Comment:
same is true for  `UINT16` (`unsigned short`) which requires `PRIu16`

Reply:
ok